api/v1alpha1/clusterextension_types.go

```diff
@@ -142,7 +141,6 @@ func init() {
 		ReasonHasValidBundleUnknown,
 		ReasonUnpackPending,
 		ReasonUnpackSuccess,
```

Comment:
here too: I think `ReasonUnpack[Pending|Failed|Success]` can go away too?

Reply:
Explained in https://github.com/operator-framework/operator-controller/pull/928#discussion_r1638192288. Success and Failed are still needed. Pending can be removed, but then there is an issue with the tests which is explained in #929

Reply:
👍